### DISCUSSION TITLE
Exclude columns of Time datatype during Automatic Scan of All Columns

### DISCUSSION THREAD

**Discussion by FrijolesNegros** [sql-bi/Bravo#533] [Ideas]
During the Automatic Scan of All Columns, perhaps it would be helpful to exclude columns of Time datatype? This would prevent generation of a calendar table that is erroneously large.

This is not critical since the user can ignore any Time columns by explicitly selecting the columns to consider.

But in an unknown model, or in a larger model with multiple columns, this might be a nice enhancement.

**Comment by albertospelta** [comment]
You are right, Time columns should be excluded from automatic scanning. The Dax.Template library already applies this exclusion during the auto scan while the Bravo UI does not as the Time columns are still displayed in the list of columns to consider even though they should not be shown.

I am going to open a new issue for that as it's probably a bug. Thanks @FrijolesNegros 

See: https://github.com/sql-bi/Bravo/issues/535
